### PR TITLE
CHANGE the usage of circleci to GitHub Actions

### PR DESCRIPTION
**Pull request by ndelangen**
Use the action we created

Not intended to me merged as is, as GitHub Actions aren't mainstream yet, but they likely will be in the future, also a few images need to be created.

Another outstanding question I have is how to I change the actual LSB-code to reflect this change?

What's the process for updating all those commits & keeping that in sync with this PR?

### CODE REVIEW

File: content/design-systems-for-developers/react/en/test.md [hunk 1]
```diff
@@ -87,34 +87,23 @@ Yikes! That small tweak resulted in a flood of UI changes.
 
 Visual testing helps identify UI changes in Storybook. Review the changes to confirm whether they’re intentional (improvements) or unintentional (bugs). If you’re fond of the new font-size, go ahead and accept the changes and commit to git. Or perhaps the changes are too ostentatious, go ahead and undo them.
 
-Let’s add visual testing to the continuous integration job. Open `.circleci/config.yml` and add the test command.
+Let’s add a second CI job for visual testing. Create `.github/workflows/chromatic.yml` and set it's content to:
```
Comment (domyen) [inline content/design-systems-for-developers/react/en/test.md]: "it's" --> "its"

File: content/design-systems-for-developers/react/en/review.md [hunk 2]
```diff
@@ -19,38 +19,29 @@ Bugs are an existential risk for design systems so we’ll do everything to prev
...
+    - uses: actions/checkout@v1
+    - run: yarn install
+    - run: yarn test
 ```
 
-At the moment, this runs `yarn test`, which is a basic React test that was set up by create-react-app for us. Let’s check that it runs on Circle:
+At the moment, this runs `yarn test`, which is a basic React test that was set up by create-react-app for us. When we commit this GitHub will run the test for us and report in the :
 
-![First build on CircleCI](/design-systems-for-developers/circleci-first-build.png)
+{ insert image of First build on GitHub }
```
Comment (domyen) [inline content/design-systems-for-developers/react/en/review.md]: Can you add a screenshot for this?

**Comment by tmeasday** [comment]
> Another outstanding question I have is how to I change the actual LSB-code to reflect this change?

> What's the process for updating all those commits & keeping that in sync with this PR?

Sorry I see you made this PR after all (after my comment on https://github.com/chromaui/learnstorybook-code/pull/20#event-2737546750

We'd need to rebase to get this change in. Luckily the Chromatic chapter is kind of late in the piece ;)

**Comment by jonniebigodes** [comment]
@ndelangen and @tmeasday and @domyen and @kylesuss i was checking this and brought to my attention that not only the documentation but the repo itself needs a change for this. I was checking and it might be a good idea to add the chromatic key to a secret to prevent misuse and probably invalidate the current one. As the site and repo is gainining even more traction everyday. Probably something like [this](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) when it comes out of public beta?

**Comment by ndelangen** [comment]
@jonniebigodes secrets don't get passed to forked PRs unfortunately

**Comment by jonniebigodes** [comment]
@ndelangen @domyen @tmeasday with the recent work that's been done with the tutorial, this pull request has become a bit outdated. I'm going to close this if all of you don't mind.

**Pull request closed without merging**